go-controller/pkg/ovn/controller/network_qos/network_qos.go

```diff
@@ -325,6 +325,10 @@ func (c *Controller) networkManagedByMe(networkSelectors crdtypes.NetworkSelecto
 		switch networkSelector.NetworkSelectionType {
 		case crdtypes.DefaultNetwork:
 			return c.IsDefault(), nil
+		case crdtypes.PrimaryUserDefinedNetworks:
+			return c.IsPrimaryNetwork(), nil
+		case crdtypes.SecondaryUserDefinedNetworks:
+			return c.IsSecondary(), nil
```

Comment:
should we check `networkSelector.SecondaryUserDefinedNetworkSelector.NamespaceSelector` and `networkSelector.SecondaryUserDefinedNetworkSelector.NetworkSelector` here?

Reply:
yes, we definitely should. 
Fixed.